### DISCUSSION TITLE
Depricated Api

### DISCUSSION THREAD

**Discussion by ghost** [prashasth-nair/AnimePeak#22] [General]
Will you refactor some depricated api such as Async task and some of the part of exoplayer in near future ? This can be helpful for improving performance.

**Comment by prashasth-nair** [comment]
Yes I am currently working on replacing All the depricated api

**Reply by ghost** [reply]
That's great news.
i wanted to connect with you for asking the question below. Dont know how to get in touch with you so i ended up creating discussions.

Can I use your code for my own project ?

**Reply by prashasth-nair** [reply]
Ya sure, you can use my code for your project. If you need any help with project I will be happy to help. You can contact me in mail: prashasthnair@gmail.com

**Comment by ghost** [comment]
Thank you for your permission.
